src/plugins/intel_cpu/tests/unit/graph/memory_state.cpp

```diff
@@ -78,26 +78,34 @@ TEST(MemStateGraphTest, smoke_Check_Memory_Modification_Guard) {
             nodes_set.insert(child);
         };
 
-        //create graph context
+        // create graph context
 
         Config conf;
         conf.rtCacheCapacity = 0;
-        auto context = std::make_shared<GraphContext>(conf, nullptr, false);
+        std::shared_ptr<NetworkMemoryControl> networkMemoryControl = std::make_shared<NetworkMemoryControl>();
```

Comment:
It looks like this variable is not used.

Reply:
cleaned up